src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -2498,7 +2498,14 @@ const int Matcher::min_vector_size(const BasicType bt) {
       // To support vector api load/store mask.
       return MaxVectorSize / 8;
     }
-    return max_size;
+    // Limit the vector size to 8 bytes
+    int size = 32 / type2aelembytes(bt);
+    if (bt == T_BYTE) {
+      // To support vector api shuffle/rearrange.
+      size = 8;
```

Comment:
Any reason that the min vector size for `"shuffle/rearrange"` is `8` while not `32/64 = 4` ?